pkg/proxy/server.go

```diff
@@ -74,15 +74,10 @@ func RunServer(ctx context.Context, cfg Config, rr Runner) error {
 	// ---------------------------------------------------------------------- //
 	// LoadTest reports
 	// ---------------------------------------------------------------------- //
-
-	// we need to do redirects because we serve html reports
-	redirectReportHandler := func(w http.ResponseWriter, r *http.Request) {
-		// FileServer removes end slashes from the path and redirects to .. - as a workaround we changed dashboard index.html to main.html
-		url := fmt.Sprintf("%s/main.html", r.URL.Host+r.URL.Path)
+	r.Get("/load-test/{id}/report", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Keep in mind that changing way of exposing reports will require a change in Kangal backend providers too. So this version should pin version of kangal-jmeter at least.

Reply:
I change the kangal-jmeter image to be BC